README.md

```diff
@@ -58,7 +55,7 @@ The core models in the OLMo family released so far are (all trained on the [Dolm
 URLs to checkpoints at intermediate steps of the models' trainings can be found in the csv files under [`checkpoints/official/`](https://github.com/allenai/OLMo/blob/main/checkpoints/official). These 'directory' URLs cannot currently be directly accessed, but files within the directory are publicly accessible. These URLs can also be provided to the training script to resume training from the checkpoint (see [Training](#training)). Each checkpoint directory consists of:
 
 - `config.yaml`: the config at that training step.
-- `model.pt`, `optim.pt`, `train.pt`: model, optimizer and training state at that training step.
+- `model.safetensors`, `optim.safetensors`, `train.pt`: model, optimizer and training state at that training step.
```

Comment:
`train.safetensors`? Also, for the original model we just have `*.pt` so we should have that format mentioned somewhere.

Reply:
We are going to save in .safetensors starting from OLMo-2

Reply:
Sure, but people might still try to use older OLMo models. The documentation should be backwards-compatible?

Reply:
No, not backwards compatible